pkg/registry/core/shoot/strategy.go

```diff
@@ -166,6 +171,25 @@ func removeForbiddenFinalizers(shoot *core.Shoot) {
 	shoot.Finalizers = finalizers
 }
 
+func removeDuplicateExtension(shoot *core.Shoot) {
+	if len(shoot.Spec.Extensions) > 1 {
```

Comment:
```
	if len(shoot.Spec.Extensions) < 2 {
            return
         }
```
to avoid nesting

Reply:
This doesn't reduce anything. I will keep the current implementation.

Reply:
It's not very important, but it does, right? All the L176-L189 will be indented one level less.

Reply:
🤷🏻